net/shadowsocks-libev/files/firewall.ss-rules

```diff
@@ -1,2 +1,7 @@
 #!/bin/sh
-/etc/init.d/shadowsocks-libev reload
+if /etc/init.d/shadowsocks-libev running; then
+    /etc/init.d/shadowsocks-libev reload
+else
+    /usr/bin/ss-rules -f
+    /usr/bin/ss-rules -6 -f
+fi
```

Comment:
`service_running` at the moment only checks a single instance's running status.  To be more precise, just check value of uci option `shadowsocks-libev.ss_rules.disabled`.

The `ss-rules` flush call should be not needed.  In normal case `firewall.ss-rules` will only be invoked by firewalld to populate ss-rules, i.e. the flush is already done.

Reply:
Maybe the service is stopped by user with command `service shadowsocks-libev stop`, shoud it be reloaded then?

Reply:
The way it was written is intended to make state of uci config to have the final say on which instances are to be run.  Constantly syncing that information to procd should make the integration more resilient while to a degree being less efficient (redundant).

If we look at the "reload" call alone, the default is to "stop then start" or just "start".  In this pull request, it's firewall.ss-rules we are talking about, it could be any other scripts.  This is how it was supposed to work.  If we are to indicate to other players that the shadowsocks-libev instances should not be run, we should make it clear.  Runtime status of procd service instances is not for this.